src/shared/components/community/community-form.tsx

```diff
@@ -129,7 +130,7 @@ export class CommunityForm extends Component<
                 onInput={linkEvent(this, this.handleCommunityNameChange)}
                 required
                 minLength={3}
-                pattern="[a-z0-9_]+"
+                pattern={validActorRegexPattern}
```

Comment:
While the frontend is preventing the use of uppercase letters for community names, `is_valid_actor_name` on the backend, which is also used for checking community names on creation, still allows them. If preventing uppercase characters is a requirement, a backend change will be needed.

Reply:
Yep that should be done also, although I do see this unique index on the community table.

`"idx_community_lower_actor_id" UNIQUE, btree (lower(actor_id::text))`

Reply:
Out of curiosity, why is that a btree? Are actor IDs organized in a tree structure?

Reply:
It's the default postgres index type, you can read more about the others here: https://www.postgresql.org/docs/current/indexes-types.html